src/c/runtime/advice_chrome_tracing.cpp

```diff
@@ -318,6 +339,20 @@ int advice_chrome_tracing_t::set_perfflow_instance_path(
     return setenv("PERFFLOW_INSTANCE_PATH", path.c_str(), 1);
 }
 
+int advice_chrome_tracing_t::set_metrics_var(const std::string value)
+{
+    return setenv("CPU_MEM_USAGE", value.c_str(), 1);
+}
+
+const std::string advice_chrome_tracing_t::get_metrics_var()
+{
+    if (NULL == getenv("CPU_MEM_USAGE"))
+    {
+        return "False";
+    }
+    return getenv("CPU_MEM_USAGE");
```

Comment:
I think instead of having this be its own environment variable, we should follow what's in `cannonicalize_perfflow_options()`, where the `cpu-mem-usage` flag would be one of the fields parsed as part of the env variable `PERFFLOW_OPTIONS`.